electron/index.js

```diff
@@ -314,17 +320,51 @@ ipcMain.on('upgrade', (event, installerPath) => {
   app.quit();
 });
 
-ipcMain.on('check-for-updates', () => {
+ipcMain.on('check-for-updates', (event, autoDownload) => {
+  // Prevent downloading the same update multiple times.
+  if (!keepCheckingForUpdates) {
+    return;
+  }
```

Comment:
What happens if I decline an update for 0.52.3, and a new version is pushed for 0.52.4? I find out on reload/restart? Might be fine. Possible to compare the version to the already downloaded version?

Reply:
I'm good with that.